packages/client/hmi-client/src/services/goLLM.ts

```diff
@@ -98,7 +98,7 @@ export async function compareModels(modelIds: string[]): Promise<CompareModelsRe
 	});
 
 	try {
-		const response = await API.get<TaskResponse>('/gollm/compare-models', {
+		const response = await API.get<TaskResponse>(`/gollm/compare-models`, {
```

Comment:
Why this was necessary?

Reply:
It was probably from my backing out inserting `?project-id=${projectId}` and the quotes did not change... fixing